git-extra/git-update-git-for-windows

```diff
@@ -211,7 +211,16 @@ update_git_for_windows () {
 	test -n "$quiet" && test "x$recently_seen" = "x$latest" && return
 
 	version=$(git --version | sed "s/git version //")
-	echo "Git for Windows $version (${bit}bit)" >&2
+	version_string="Git for Windows $version (${bit}bit)"
+	filename_for_download="$bit\-bit\.exe"
+
+	if [ -d "/arm64/bin" ]
```

Comment:
Technically, the Git wrapper does more: it first verifies that we _are_ running on ARM64. Therefore, we could implement the `--is-running-on-ARM64` option in `/cmd/git.exe` and use that. But that sounds like quite a bit of effort for almost no value in return, so I'm fine with the directory check.

To align with Git's coding style, I would like to use `test` instead of `[`, though.